src/three_by_three_board.py

```diff
@@ -1,7 +1,20 @@
+import src.constants as constants
+
 class ThreeByThreeBoard():
+    # => All of the possible winning combinations
+    WIN_COMBINATIONS = [
+        [0, 1, 2], # Top row
+        [3, 4, 5], # Middle row
+        [6, 7, 8], # Bottom row
+        [0, 3, 6], # First column
+        [1, 4, 7], # Second column
+        [2, 5, 8], # Third Column
+        [0, 4, 8], # Left to Right diagonal
+        [2, 4, 6]  # Right to Left diagonal
+    ]
```

Comment:
Just to get my thoughts out here - in my head there seems to be arguments for and against - but does the board need to understand how it is won? Could it be another class, such as a Rules class that would know that? I'm not sure if that would constitute feature envy - or if it's a valid separation of concerns.

Reply:
It would seem to me the game class would want to know that more than the board - since the game class is just trying to know whether or not the game should end, and which message it should display.

That might mean this should live as its own thing.

Reply:
Also - Is there some way you can turn those comments into variable names? That might start to show the idea that this board class is doing an aweful lot. what would happen if we were to change the size of the board? or change the rules to not allow diagonal wins, or getting four corners?